### PR TITLE
DDO-11

### PR DESCRIPTION
**Pull request by SrishtiLodhi**

### CODE REVIEW

File: package/extensions/page-break.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+              const node = state.doc.nodeAt(pos);
+
+              console.log(node?.type.name === 'pageBreak', 'name of node');
+              if (node?.type.name === 'pageBreak') {
+                const tr = state.tr;
+                const nodeSize = node.nodeSize;
+
+                // Log nodeSize and pos for debugging
+                console.log(
```
Comment (kylengn) [inline package/extensions/page-break.ts:135]: remove

File: package/extensions/page-break.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    return [
+      new Plugin({
+        key: new PluginKey('pageBreakPlugin'),
+        props: {
+          handleClick: (view, pos, event) => {
+            const target = event.target as HTMLElement;
+
+            // Check if the clicked element is the image to remove the page break
+            if (target.classList.contains('remove-page-break-icon')) {
+              console.log('Remove page break image clicked');
```
Comment (kylengn) [inline package/extensions/page-break.ts:124]: remove

File: package/extensions/page-break.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                  nodeSize,
+                );
+
+                tr.delete(pos, pos + nodeSize);
+
+                dispatch(tr.scrollIntoView());
+
+                return true;
+              }
+            } else {
```
Comment (kylengn) [inline package/extensions/page-break.ts:148]: remove else if not having any operation

**Review by kylengn — CHANGES_REQUESTED** [review]

File: package/extensions/page-break.ts (hunk 1 above)
Comment (SrishtiLodhi) [reply package/extensions/page-break.ts]: I have removed this

File: package/extensions/page-break.ts (hunk 3 above)
Comment (SrishtiLodhi) [reply package/extensions/page-break.ts]: removed it 👀

File: package/extensions/page-break.ts (hunk 2 above)
Comment (SrishtiLodhi) [reply package/extensions/page-break.ts]: Removed the console

**Review by kylengn — APPROVED** [review]

**Pull request merged**